web/src/test/java/com/garret/movies/MovieApplicationTest.java

```diff
@@ -3,14 +3,19 @@
 import lombok.extern.slf4j.Slf4j;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.modelmapper.ModelMapper;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.test.context.junit4.SpringRunner;
 
 @SpringBootTest(classes = MovieApplication.class)
 @Slf4j
 @RunWith(SpringRunner.class)
 public class MovieApplicationTest {
 
+    @MockBean
+    private ModelMapper modelMapper;
```

Comment:
@Garret-vin This test checks if your context is OK. But you mock bean here... This means that your context might not be OK ))

Reply:
@Dominys I made it temporarily as a stub